tests/unittests/ThreadPoolExecutorTest.cpp

```diff
@@ -155,13 +165,14 @@ class TestDeviceManager final : public runtime::DeviceManager {
   }
 
 private:
-  /// This struct wraps all of the data needed to reply to a runFunction() call.
-  /// It exists so that that all of these things can be stored in one map.
+  /// This struct wraps all of the data needed to reply to a runFunction()
```

Comment:
This/the rest of the comments didn't actually change, right? `clang-format` issue?

Reply:
Yeah just the auto clang-formatter, sorry for the noise.